test/unit/app/controllers/permissions/helpers.js

```diff
@@ -0,0 +1,65 @@
+import assert from 'assert'
```

Comment:
The default export of `assert` [has been deprecated](https://nodejs.org/docs/latest-v10.x/api/assert.html#assert_legacy_mode) in favour of "strict mode", i.e. this:
```suggestion
import { strict as assert } from 'assert'
```

Reply:
The same goes for the other tests where this is used

We use `assert` all over the place but I've been trying to stick to the strict one in new tests

Reply:
Huh, neat.